napari/view_layers.py

```diff
@@ -111,6 +114,7 @@ def view_image(
         blending=blending,
         visible=visible,
         path=path,
+        axis_labels=axis_labels,
```

Comment:
as noted above I don't think this should go on the `add_image` method, but it should go a few lines above 
```python
viewer = Viewer(title=title, ndisplay=ndisplay, order=order, axis_labels=axis_labels)
```
You'll also need to change the function signature of `Viewer` in `viewer.py` to be
```python
    def __init__(self, title='napari', ndisplay=2, order=None, axis_labels=None):
```
and inside the `ViewerModel` in `components/viewer_model.py` to be
```python
    def __init__(self, title='napari', ndisplay=2, order=None, axis_labels=None):
```